### PR TITLE
Refactored MySQL(2) Node Example Drivers

### PR DESCRIPTION
**Pull request by sm7x**

### CODE REVIEW

File: signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const initSpan = tracer.startSpan('initDB');
+
+  // Using the Scope's activate() function, we establish
+  // initSpan as the active span for all asynchronous context
+  // stemming from this anonymous function.
+  tracer.scope().activate(initSpan, () => {
+    //    pool = mysql2.createPool(config.poolConfig)
+    //    poolPromise = pool.promise();
+
+    tracer.scope().activate(initSpan, () => {
```
Comment (rmfitzpatrick) [inline signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js]: Are this second activation and above comments intentional?  You are also finishing the initSpan immediately after this, which probably shouldn't happen until the table is successfully created.

Comment (sm7x) [reply signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js]: Thanks for catching that. Updated it.

File: signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                     date TIMESTAMP NOT NULL DEFAULT current_timestamp,
+                     completed TINYINT(1) NOT NULL DEFAULT 0
+                     )`;
+
+      connection.query(createTodos, (error, results, fields) => {
+        if (error) {
+          if (error.code === 'PROTOCOL_CONNECTION_LOST') {
+            console.error('error.message: ' + error.message);
+            initSpan.setTag('reconnecting', true);
+            initDB();
```
Comment (rmfitzpatrick) [inline signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js]: Mind waiting 5s before this?

File: signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          } else {
+            initSpan.setTag('error', true);
+            initSpan.log({'error': error.message});
+            console.error('error.message: ' + error.message);
+          }
+        }
+        try {
+          connection.release();
+        } catch (error) {
+          initSpan.log({'error': error.message});
```
Comment (rmfitzpatrick) [inline signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js]: error tag as well

**Pull request merged**